lib/evil_seed/refinements/in_batches.rb

```diff
@@ -7,9 +7,7 @@ module Refinements
     # See https://github.com/rails/rails/commit/25cee1f0373aa3b1d893413a959375480e0ac684
     # The ActiveRecord MIT license is obviously compatible with our license (MIT also)
     module InBatches
-      refine ActiveRecord::Relation do
-        using EvilSeed::Refinements::InBatches
-
+      refine ActiveRecord::Batches do
```

Comment:
This makes MRI 2.3 to fail as it doesn't want to refine modules

Reply:
Anyway, it didn't help(